understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>意図</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make logins possible with less mental effort.</dd>
+                  <dt>目標</dt>
+                  <dd>より少ない精神的労力でログインできるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Don't make people solve, recall, or transcribe something to log in.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ログインするために人々に何かを解かせたり、思い出させたり、転記させたりしない。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with cognitive disabilities cannot solve puzzles, memorize a username
-                     and password, or retype one-time passcodes.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>認知障害のある人々の中には、パズルを解くことも、ユーザ名及びパスワードを記憶することも、ワンタイムパスコードを再入力することもできない人もいる。
```

Comment:
「A も B も C も、できない人もいる」ですと、A、B、C いずれもできない人、のように読み取れてしまいそうですが、原文では or が用いられています。以下、いかがでしょうか？

```suggestion
                  <dd>認知障害のある人々の中には、パズルを解くこと、ユーザ名及びパスワードを記憶すること、あるいはワンタイムパスコードを再入力することができない人もいる。
```

Reply:
ありがとうございます。ここのorは又はなのでそのようにして修正します。